v2/pkg/catalog/loader/loader.go

```diff
@@ -376,3 +379,49 @@ func workflowContainsProtocol(workflow []*workflows.WorkflowTemplate) bool {
 	}
 	return false
 }
+
+func (store *Store) GetUncoverTargetsFromMetadata(delay, limit int) chan string {
```

Comment:
we should remove this function from loader and instead move it to a `pkg/protocols/common/uncover` package where relevant code should be placed by refactoring current implementation

Reply:
moved this code to new package